### PR TITLE
fixing the beginners mistake

### PR DESCRIPTION
**Pull request by Sparkezel**
## About The Pull Request
When I added jamming to mosins, something got fucked with spells based of them (blood bolt/arcane barrage/lesser summon guns). Thanks to @necromanceranne for fiding the solution for that

## Why It's Good For The Game
bug fixes good

## Changelog
:cl:
fix: Spells like lesser summon guns, blood bolt and arcane barrage work again properly.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
speedmerge, I love lesser summon guns

File: code/modules/projectiles/guns/ballistic/rifle.dm [hunk 1]
```diff
@@ -96,7 +96,7 @@
 				jammed = TRUE
 			jamming_chance  += jamming_increment
 			jamming_chance = clamp (jamming_chance, 0, 100)
-	..()
+	. = ..()
```
Comment (Jared-Fogle) [inline code/modules/projectiles/guns/ballistic/rifle.dm:99]:
```suggestion
	return ..()
```

Comment (cacogen) [reply code/modules/projectiles/guns/ballistic/rifle.dm:99]: What's the difference between these? I always thought they meant the same thing. But I know now one calls the parent proc on the line it's on and the other returns early and calls the parent proc. I don't know what the . means.

Comment (Jared-Fogle) [reply code/modules/projectiles/guns/ballistic/rifle.dm:99]: They mean the same thing in this specific case, it's just `return` at the end is more clear.

Comment (Sparkezel) [reply code/modules/projectiles/guns/ballistic/rifle.dm:99]: ah, alright

**Review by Mothblocks — APPROVED** [review]

File: code/modules/projectiles/guns/ballistic/rifle.dm (hunk 1 above)
Comment (cacogen) [reply code/modules/projectiles/guns/ballistic/rifle.dm]: I thought return ..() stopped the rest of the code running while . = ..() just ran it after that line

**Pull request merged**